pyproject.toml

```diff
@@ -38,14 +38,14 @@ torch = [
   {markers = "sys_platform == 'linux' and platform_machine == 'x86_64'", url="https://download.pytorch.org/whl/cpu/torch-1.13.1%2Bcpu-cp310-cp310-linux_x86_64.whl"},
   {markers = "sys_platform == 'win' and platform_machine == 'amd64'", url="https://download.pytorch.org/whl/cpu/torch-1.13.1%2Bcpu-cp310-cp310-win_amd64.whl"},
 ]
-jinja2 = "^3.1.2"
-celery = "^5.2.7"
-redis = "^4.5.4"
-ctranslate2 = "4.2.1"
-aiofiles = "^23.2.1"
+jinja2 = "^3.1.4"
+celery = "^5.4.0"
+redis = "^5.2.0"
+ctranslate2 = "4.4.0"
```

Comment:
this is the one giving me problems now 🤔

looks like it's missing a mac x86_86 package 🤔🤔🤔

4.3.1 has x86_64 and nudging this down to that version fixes my run of `poetry install`